api_tests/package.json

```diff
@@ -6,6 +6,9 @@
     "scripts": {
         "test": "mocha --bail"
     },
+    "engines": {
+        "node": "11.10"
```

Comment:
This makes `yarn` fail if it would execute against a node version that is `< 11.10` or `> 11.11`